openshift/Tailorfile

```diff
@@ -1,7 +1,6 @@
 namespace prov-dev
 selector app=prov-app
-param-file ../../../ods-configuration/ods-core.env
-param-file ../../../ods-configuration/ods-provisioning-app-dev.env
+param-file ods-provisioning-app.env
```

Comment:
Should that file be gitignored?

Reply:
No need to add it as *.env files are already ignored.

https://github.com/opendevstack/ods-provisioning-app/blob/3d9370053b54fee2961bd90cf47a5ec53e7bd385/.gitignore#L33